src/gui/transferlistmodel.h

```diff
@@ -93,6 +97,9 @@ class TransferListModel : public QAbstractListModel
 
     BitTorrent::TorrentHandle *torrentHandle(const QModelIndex &index) const;
 
+    void setStateForeground(enum BitTorrent::TorrentState state, const QColor& color);
```

Comment:
probably you don't need the `enum` here and below and in .cpp

Reply:
It's optional in C++ and we usually omit it.

Reply:
done.